homeassistant/components/starline/config_flow.py

```diff
@@ -212,7 +213,7 @@ async def _async_authenticate_user(self, error=None):
                 self._captcha_image = data["captchaImg"]
                 return self._async_form_auth_captcha(error)
 
-            raise Exception(data)
+            raise HomeAssistantError(data)
```

Comment:
We should catch the exception in the step methods, and probably abort with unknown error reason.

Reply:
Same here, revert and added ` pylint: disable`.

The  exception is handled directly in the next line already.
https://github.com/home-assistant/core/blob/d017214d8b2a71728a7859703b6a47db56a5e498/homeassistant/components/starline/config_flow.py#L215-L218